doc/modules/preprocessing.rst

```diff
@@ -698,16 +688,13 @@ It is possible to adjust the threshold of the binarizer::
          [1., 0., 0.],
          [0., 0., 0.]])
 
-As for the :class:`StandardScaler` and :class:`Normalizer` classes, the
-preprocessing module provides a companion function :func:`binarize`
-to be used when the transformer API is not necessary.
```

Comment:
Let's keep this and just remove the mention to StandardScaler. it's still valuable to mention `binarize` and `normalize` since there's no risk of data leak with these ones

Reply:
Ok